scripts/haproxy/haproxy_osver8.cfg

```diff
@@ -135,6 +135,11 @@ frontend main
     acl s3authbackendacl dst_port 9080
     use_backend s3-auth if s3authbackendacl
 
+    # Add a Request ID to request header and haproxy log
+    unique-id-format        %{+X}o\ %ci:%cp_%fi:%fp_%Ts_%rt:%pid
+    unique-id-header        Request_id
```

Comment:
client may also send such headers, may be we can change this to some uncommon value

Reply:
Will be changed